areaofcircle.py

```diff
@@ -0,0 +1,12 @@
+class AreaOfCirlce:
```

Comment:
As discussed, modify this to create Circle class.

Reply:
Changed the class name